tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -56,6 +59,21 @@ describe('Lobby - Page Content', () => {
   });
 });
 
+describe.only('Lobby - Page Content (Ranked)', () => {
+  beforeEach(() => {
+    setup(true);
+  });
+
+  it('Displays ranked header', () => {
+    cy.contains('h1 small', 'Ranked');
+    cy.log(cy.window().its('cuttle.app.$store.state.gameList.games'));
```

Comment:
Is this log useful? Wondering if it's stale from tinkering stages or we want it. I think it's funky to log the list of games rather than the current one, but it's my same point about moving the `isRanked` check into the `game` module

Reply:
Nope, just missed it while I was debugging the issue to try and find where the isRanked property was from. This is why I like code review, this 'works' but is not ideal and the 2nd/3rd iterations will be much better for the effort 👍🏼